internal/k8shandler/logstore_lokistack_test.go

```diff
@@ -0,0 +1,162 @@
+package k8shandler
```

Comment:
I believe using semantic.DeepEqual makes writing the following testcases obsolete

Reply:
Seems I forgot about `equality.Semantic` yesterday, but I think I would keep the tests anyway to make it clear what we are actually comparing.